### PR TITLE
Add meeting notes for 2020-09-29

### PR DESCRIPTION
**Pull request by dschuff**
sorry for the delay on this.
@yuri91 @rossberg @conrad-watt please post slide links

### CODE REVIEW

**Comment by conrad-watt** [comment]
https://docs.google.com/presentation/d/1-ajjGZpjAiGYOJlwswij9Mq6YGltmuELg3tbhu30VrI/edit?usp=sharing

**Comment by yuri91** [comment]
https://drive.google.com/file/d/1LGqAVJYjBnFbhx2ANUuuA9YMELHI6gEw/view?usp=sharing

File: main/2020/CG-09-29.md [hunk 1]
```diff
@@ -41,4 +41,253 @@ Installation is required, see the calendar invite.
...
+| 13 | 16 | 4 | 0 | 0 |
+
+Poll passes
+
+switching presenter; meanwhile:
+
+### Announcement from Ross Tate:
+
+RT: SOIL Initiative is starting a [seminar series](http://soil-initiative.org/#seminar). First one is Monday October 12 at 12 PM EDT/9 AM PDT
+Then next one on Oct 23 at 4PM
```
Comment (taralx) [inline main/2020/CG-09-29.md:120]: Timezone?

File: main/2020/CG-09-29.md [hunk 2]
```diff
@@ -41,4 +41,253 @@ Installation is required, see the calendar invite.
...
+
+AR: I'm not aware of any problems with use of bottom type here?
+
+DS: Before we go into more general issues, I'm hearing agreement on this particular tweak; we should go ahead with unanimous consent, any objections?
+
+<no objections>
+
+### Add type for unreachable states (https://github.com/WebAssembly/design/issues/1379) (Conrad Watt and Ross Tate)
+
+<TODO(CW): insert slides>
```
Comment (dschuff) [inline main/2020/CG-09-29.md:215]:
```suggestion
CW [presenting](https://docs.google.com/presentation/d/1-ajjGZpjAiGYOJlwswij9Mq6YGltmuELg3tbhu30VrI/edit?usp=sharing)
```

File: main/2020/CG-09-29.md [hunk 3]
```diff
@@ -41,4 +41,253 @@ Installation is required, see the calendar invite.
...
+### Announcement from Ross Tate:
+
+RT: SOIL Initiative is starting a [seminar series](http://soil-initiative.org/#seminar). First one is Monday October 12 at 12 PM EDT/9 AM PDT
+Then next one on Oct 23 at 4PM 
+
+First up is Thomas Lively on module splitting
+
+## Presentation and feedback-gathering on branch hinting ([issue](https://github.com/WebAssembly/design/issues/1363)) (Yuri Iozzelli)
+
+TODO(YI): Link to slides
```
Comment (dschuff) [inline main/2020/CG-09-29.md:126]:
```suggestion
Yuri Iozzelli [presenting](https://drive.google.com/file/d/1LGqAVJYjBnFbhx2ANUuuA9YMELHI6gEw/view?usp=sharing)
```

**Comment by dschuff** [comment]
(I merged, but it would still be good to get a presentation link from @rossberg )

**Pull request merged**